src/components/Footer.js

```diff
@@ -1,9 +1,17 @@
+import Link from 'next/link';
+
 export default function Footer() {
 	return (
 		<>
-			<button type="button">My Contact</button>
-			<button type="button">Add contact</button>
-			<button type="button">Contact list</button>
+			<Link type="button" href="/profile/my-contact">
+				<button type="button">My Contact</button>
+			</Link>
+			<Link type="button" href="/add-contact">
+				<button type="button">Add contact</button>
+			</Link>
+			<Link type="button" href="/contact-list">
+				<button type="button">Contact list</button>
+			</Link>
 		</>
 	);
 }
```

Comment:
Your footer is simply a navigation. Why don't you just call the component navigation? That would be semantically more correct.